src/main/java/user/delete_comment/UserOwnCommentVerifier.java

```diff
@@ -0,0 +1,33 @@
+package user.delete_comment;
+
+import user.delete_comment.exceptions.NotOwnCommentException;
+import entity.Comment;
+
+public class UserOwnCommentVerifier {
+
+    private final Comment comment;
+
+    private final String userId;
+
+    public UserOwnCommentVerifier(Comment comment, String userId){
+        this.comment = comment;
+        this.userId = userId;
+    }
+
+    public Comment getComment(){
+        return this.comment;
+    }
+
+    public String getUserId(){
+        return this.userId;
+    }
+
+    public void verify() throws NotOwnCommentException{
+        String userIdGiven = this.getUserId();
+        String userIdComment = this.getComment().getUserID();
+        if (userIdGiven.equals(userIdComment)) {
```

Comment:
is this not equals?

Reply:
I'm comparing strings here, so I think I should use .equals

Reply:
So if the user is matched the user from the comment, they do not own the comment?